### DISCUSSION TITLE
What kind of proposals are going to be moved to discussions?

### DISCUSSION THREAD

**Discussion by Xrayez** [godotengine/godot-proposals#3982] [General Discussions]
What set of criteria is going to be used to determine what kind of proposals should be migrated?

Note that most proposals don't even mention the concrete project they are working on. Should those proposals be moved immediately? Clearly, there are proposals that aim to improve existing engine features (enhancement proposals), so it's not always necessary to describe the project there, I'm only talking about new feature proposals that don't describe the project they are working on.

Likely most [`meta` proposals](https://github.com/godotengine/godot-proposals/issues?q=is%3Aopen+is%3Aissue+label%3Ameta) could be moved to discussions.

See also:
- https://github.com/godotengine/godot-proposals/issues/39
- https://github.com/godotengine/godot-proposals/issues/47

**Comment by YuriSizov** [comment]
All proposals that don't describe a solid case for an enhancement and don't provide any technical detail to how the implementation should work will be closed and suggested to be moved to discussions. Think GIP == RFC.

> Likely most [`meta` proposals](https://github.com/godotengine/godot-proposals/issues?q=is%3Aopen+is%3Aissue+label%3Ameta) could be moved to discussions.

Yes indeed. Feel free to move any proposals that you've made before to a discussion ahead of us!

**Reply by Xrayez** [reply]
> Yes indeed. Feel free to move any proposals that you've made before to a discussion ahead of us!

Users cannot move their own issues to discussions, only maintainers can.

**Reply by YuriSizov** [reply]
Sure, but you don't have to actually move the issue, you can just close the proposal issue and open a discussion referencing it. It would probably end up being less messy this way.

**Reply by Xrayez** [reply]
It's going to be difficult to track what kind of issues has been moved, as GitHub won't be able to redirect closed issues to discussions automatically. GIP exists for more than 2 years now and it's better not to "break" existing links, as users may continue discussion in the old issues.

**Reply by YuriSizov** [reply]
We've discussed it, and it makes sense to move smaller issues with not much comments in them, but doesn't work well for longer issues. So for those we'd suggest opening a new thread and making a concise summary of the deliberations so far.

**Reply by GeorgeS2019** [reply]
@Xrayez @pycbouh  => I think it is better to trim GIP by simply copy the text of **GIP (less developed)** over to the discussion including the user who initiated the GIP.  After a few days, it is clear the discussion is working as the **GIP ideation**. Once someone has gathered sufficient feedback and discussion, the person can create a new GIP that reflects a well-balanced discussion as well as identifying participants who are likely to contribute to the implementation of the GIP.

**Reply by Xrayez** [reply]
@GeorgeS2019 well, someone has to do this and make the decision. Asking users to move proposals to discussions would take more work. Copying text and notifying users that their proposal have a new place at discussions will also require more work. It's also weird and bad to copy ("steal") the text from old proposals like that.

So, I think it would be best for Godot maintainers to just move all non-technical/idea proposals to discussions themselves via GitHub migration tool (which *is* easy, and I know it's easy because I've done this in https://github.com/goostengine/goost/discussions).

Also note that I don't have any kind of maintainer privileges here at **godotengine** whatsoever, so it would be up to Godot members to do this kind of work, in either case.

**Reply by GeorgeS2019** [reply]
@Xrayez  let the Godot community let go more "steam" at the newly created discussion. The c# discussion track has gathered close to 50 comments.  I see there are some optimization to organize both the GIP and the discussion.  It is a good start  Let us wait a few more days or weeks. The decision to trim is obvious. It could be as simple as giving ENOUGH notice of the decision how to do it.  The first priority is to trim away GIP which no longer of interest or without followers. As long as the GIP or issue dependencies are there, there are ways to track through the GIP priorities.  

One possibility is to organize a monthly ranked GIP. Close all GIP. Ranking of GIPs for each month is discussed, voted  at the GIP discussion. Only those GIPs with clear owners and with community backing and interest are ranked. This could also be organized into different categories e.g. 2D, 3D, etc .Each category has its own rank for voting.
